changelog.md

```diff
@@ -1,3 +1,36 @@
+# 3.8.0 / 2021-04-27
+
+### New Features
+
+- Content can be added to personal favourites
+- Add and comment a publication
```

Comment:
Is this feature the publications by themselves? Publications were added in the 3.7.0 I think.

Reply:
thats right. its a mistake in project 3.8.0 description.
I remove this part.

Reply:
done